packages/client/src/declarations/index.ts

```diff
@@ -523,7 +523,7 @@ export function createReviewDeclaration(
     id: declarationId,
     data: formData,
     duplicates,
-    originalData: formData,
+    originalData: status === 'REGISTERED' ? formData : {},
```

Comment:
```suggestion
    originalData: status !== 'IN_PROGRESS' ? formData : {},
```

Reply:
This would affect only the incomplete declaration which is what we want right?

Reply:
What about the "In Review", "Requires updates" declarations? Their statuses are "In_Progress?